src/components/notifications/SIGNL4.vue

```diff
@@ -0,0 +1,88 @@
+<template>
```

Comment:
Usually, we try to give users information how to get these fields.
I don't know how intuitive the UI for `SIGNL4` to find for example the webhook url => could you add a comment where users can find either
- the docs for the provider
- where in the UI this URL can be found

Reply:
OK. The docs will be available here: https://docs.signl4.com/integrations/uptime-kuima/uptime-kuma.html